### ISSUE TITLE
[Question] still under development?

### ISSUE THREAD

**Issue by snwflake** [varjolintu/keepassxc-proxy#3]
Same question goes for the FF extension and the fork aswell.

**Comment by varjolintu** [comment]
Yes, this is still under development. The main issues with the proxies are:
1) Rust version and Firefox don't get along, I don't know why
2) Python version cannot be hidden from the Dock with macOS

There's actually no open issues with the fork but I would like to make it compatible with multiple proxies. Now you can use only one at a time.

The extension is always under development but it's already stable so it's safe to use.

**Comment by varjolintu** [comment]
A new C++ version is now online.

**Comment by snwflake** [comment]
Thanks for the C++ version! Gonna try that aswell :)

**Comment by varjolintu** [comment]
The only thing still missing from the C++ version is the timeout operation. So currently it's not perfect, but it will be :)

So don't get furious if the popup gets stuck to the checking phase.. this only happens if KeePassXC is closed and no state can be received. This wil be fixed soon.

**Comment by varjolintu** [comment]
The timeout operation is no longer needed (the browser extension handles the timeouts) so the proxy implementation is lot more simpler. I have already a working copy with the same proxy and KeePassXC with Unix Domain sockets. It is possible that these replace the plain UDP socket under Linux/Unix/macOS.

**Comment by snwflake** [comment]
That'll be great. Currently FF needs some time to kill the proxy and shuts itself down after ~3 seconds.

**Comment by varjolintu** [comment]
Yes, I'm aware of those issues also. Sometimes the proxy process is not killed at all. Of course it doesn't take any CPU or anything but it's annoying at should not happen.

**Comment by snwflake** [comment]
Mhm...never happened to me though. Proxy is always killed for me.

**Comment by Cherkah** [comment]
a refreshg question:
still under development?
thx

**Comment by varjolintu** [comment]
@Cherkah I haven't had the time to update this repo for a while.
